Dockerfile

```diff
@@ -24,9 +26,9 @@ ADD . /cryptpad
 
 WORKDIR /cryptpad
 
-RUN npm install \
-   && npm install -g bower \
-   && bower install --allow-root
+RUN npm install
+RUN npm install -g bower
```

Comment:
`docker build` will bail in both cases, but chaining them with `&&` reduces the number of layers of the final image.